modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/pages/data_set/tabs/SortingPage.ts

```diff
@@ -45,4 +45,26 @@ export class SortingPage {
 	async openAddSortingModal() {
 		await this.addSortingButton.click();
 	}
+
+	async selectTab(tabLabel: string) {
+		await this.dataSetPage.selectTab(tabLabel);
+	}
+
+	async getTableLabelCellTexts(): Promise<string[]> {
+		const rows = await this.sortingTable.locator('tr').all();
+
+		rows.shift(); // Exclude the header
+
+		const labelTexts: string[] = [];
+
+		for (const row of rows) {
+			const labelCell = row.locator('td').nth(1);
+
+			const labelText = await labelCell.textContent();
+
+			labelTexts.push(labelText);
+		}
+
+		return labelTexts;
+	}
```

Comment:
Can we simplify this to something like
```js
locator('tr > td:first-child').allInnerTexts()
```
This can maybe be inline. If util method is more convenient, we can tweak name to something like `getFirstColumnInnerTexts()`

Reply:
Thanks! I'll simplify this. I renamed it to `getTableColumnInnerTexts(2)`, adding a parameter to get any column's text. (The first column is the drag handles)